components/resources/rewards_strings.grdp

```diff
@@ -550,4 +550,140 @@
+    Your Brave Rewards cannot be verified because you've reached the maximum verified device limit.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_DEVICE_LIMIT_REACHED_TITLE" desc="">
+    Device limit reached
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_FLAGGED_WALLET_TEXT_1" desc="">
+    Unfortunately, your Brave Rewards profile was flagged because our automated checks found some unexpected Brave Ads activity.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_FLAGGED_WALLET_TEXT_2" desc="">
+    While your profile is flagged, you may continue to see ads, but won't be eligible to receive payouts. You also won't be able to connect a custodial account to Brave Rewards.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_FLAGGED_WALLET_TEXT_3" desc="">
+    It's possible your profile was flagged in error, and no immediate action is required on your part. The flag on your profile may be removed in the future.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_FLAGGED_WALLET_TEXT_4" desc="">
+    <ph name="BEGIN_LINK">$2</ph>Learn more about Brave's process for flagging Rewards profiles.<ph name="END_LINK">$3</ph>
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_FLAGGED_WALLET_TITLE" desc="">
+    Brave Rewards profile is flagged
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_KYC_REQUIRED_TEXT" desc="">
+    Please try again after you have completed ID verification on <ph name="PROVIDER">$1<ex>Uphold</ex></ph>.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_KYC_REQUIRED_TITLE" desc="">
+    You need a verified account to log in
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_MISMATCHED_COUNTRIES_TEXT" desc="">
+    Hmm, it looks like your Brave Rewards country does not match the country of the <ph name="PROVIDER">$1<ex>Uphold</ex></ph> account you're trying to connect.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_MISMATCHED_COUNTRIES_TITLE" desc="">
+    Country mismatch
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_MISMATCHED_PROVIDER_ACCOUNTS_TEXT" desc="">
+    Hmm, it looks like your Brave Rewards has already been verified with another <ph name="PROVIDER">$1<ex>Uphold</ex></ph> account. Please try verifying again using your previous account.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_MISMATCHED_PROVIDER_ACCOUNTS_TITLE" desc="">
+    Different account
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_PROVIDER_UNAVAILABLE_TEXT_1" desc="">
+    It looks like connecting <ph name="PROVIDER">$1<ex>Uphold</ex></ph> to Brave Rewards is temporarily unavailable. Please try again later.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_PROVIDER_UNAVAILABLE_TEXT_2" desc="">
+    You can view <ph name="BEGIN_LINK">$2</ph>service status here<ph name="END_LINK">$3</ph>.
+  </message>
+  <message name="IDS_REWARDS_AUTHORIZE_PROVIDER_UNAVAILABLE_TITLE" desc="">
+    Connecting to selected custodian temporarily unavailable
```

Comment:
Connecting to selected custodian **is** temporarily unavailable

Reply:
Keep as is